backend/api/urls.py

```diff
@@ -0,0 +1,17 @@
+from django.conf.urls import url
+from rest_framework.urlpatterns import format_suffix_patterns
+
+from api import views
+
+urlpatterns = [
+    url(r'^attendance/(?P<pk>[0-9]+)/$', views.AttendanceView.as_view()),
+    url(r'^comments/(?P<pk>[0-9]+)/$', views.CommentView.as_view()),
+    url(r'^events_list/$', views.EventListView.as_view()),
+    url(r'^events/(?P<pk>[0-9]+)/$', views.EventView.as_view()),
```

Comment:
During my tests, I noticed that `/api/events/` does not work.

Reply:
I'm on it..